common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/Work.scala

```diff
@@ -198,8 +201,12 @@ object WorkFsm {
   }
 
   implicit val sourceToMerged = new Transition[Source, Merged] {
-    def state(state: Source, numberOfSources: Int): Merged =
-      Merged(state.sourceIdentifier, numberOfSources)
+    def state(state: Source, args: (Option[Instant], Int)): Merged =
+      Merged(
+        state.sourceIdentifier,
+        args._1.getOrElse(state.modifiedTime),
```

Comment:
In what instance is the modifedTime not present?

Reply:
This was for convenience if you want to transition the state without changing the `modifiedTime` (eg you use the existing one in the `Source` state) - makes life easier in tests in particular